ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js

```diff
@@ -302,13 +302,11 @@ function BlueprintService($log, $q, $sce, paletteApi, iconGenerator, dslService)
 
     function refreshConfigMemberspecsMetadata(entity) {
         let promiseArray = [];
-        if (entity.isCluster()) {
```

Comment:
Is there really another case, outside a cluster or fabric where we can have a memberspec?

Reply:
yes -- `BrooklynNode` uses it as part of upgrade, test framework uses it i think, and there is a downstream project that wants to use this in a different type of cluster.

Reply:
Ah good point, this should be removed then.